src/js/popup.js

```diff
@@ -908,6 +922,53 @@ Logic.registerPanel(P_CONTAINERS_EDIT, {
       const tr = document.createElement("tr");
       fragment.appendChild(tr);
       tr.classList.add("container-panel-row");
+      tr.draggable = true;
+      tr.dataset.containerId = identity.cookieStoreId;
+      tr.addEventListener("dragstart", e => {
```

Comment:
```suggestion
      tr.addEventListener("dragstart", (e) => {
```
Just to remain consistent with other code in this repo

Reply:
Fixed